rust/Cargo.toml

```diff
@@ -174,8 +174,8 @@ ripemd = "0.1.3"
 hyper = "0.14"
 hyper-tls = "0.5.0"
 hex = "0.4.3"
-tendermint-rpc = { version = "0.32.0", features = ["http-client", "tokio"]}
-hpl-interface = "0.0.6-rc5"
+tendermint-rpc = { version = "0.32.0", features = ["http-client", "tokio"] }
+hpl-interface = "=0.0.6-rc3"
```

Comment:
how come this is using `rc3` but run-locally uses `rc6`?

Reply:
it's a hacky approach to make runnable with this - the reason of doing like this is i realized payload in rc6 is not up to date with rc6 contract binary, so i published new version in empty slot temporarily.

Reply:
Makes sense, I'll merge it like this but let's open a follow-up PR to address when it's fixed